Automattic/newspack-popups
# fix: naming duplicated prompts

### All Submissions:






### Changes proposed in this Pull Request:



Closes #628.

### How to test the changes in this Pull Request:

1. Create a prompt from the campaigns screen with `My Prompt` as a title.
2. Duplicate the prompt (from the prompt edit screen) and notice that the suggested name is correct and should be `My Prompt copy`.
3. Duplicate again the first prompt and change its title this time to `Second Prompt`.
4. Duplicate the `Second Prompt`, and notice that the title generated `My Prompt copy 2` is based on the original prompt and not the parent prompt.
5. Check out this PR.
6. Duplicate the first prompt and change its name to `Third Prompt`.
7. Duplicate `Third Prompt` and notice that this time, the title generated `Third Prompt copy` is based on the parent prompt.
8. Duplicate `Third Prompt` or `Third Prompt copy` and notice that the suggested title `Third Prompt copy 2` is based on the parent title and the total number of occurrences.

### Other information:

All the duplicate actions should be done using the `duplicate` button from the prompt edit screen. [A separate PR](https://github.com/Automattic/newspack-plugin/pull/1326) will add the functionality to the duplicate menu in the prompts list screen.
